libs/gi/dm-localization/src/executors/gen-locale/executor.spec.ts

```diff
@@ -1,8 +0,0 @@
-import executor from './executor'
```

Comment:
https://stackoverflow.com/questions/78806479/jest-tests-started-failing-with-typeerror-a-dynamic-import-callback-was-invoked

The file is unused anyway, so I decided to remove it